fighthealthinsurance/medicaid_api.py

```
@@ -0,0 +1,928 @@
+    )
+
+
+def get_medicaid_info(query: Dict[str, Any]) -> str:
+    """
+    query example: {"state":"StateName","topic":"","limit":5}
+    Returns a clean, professional format with key contact info.
+    """
+    state_short = _normalize_state(
+        (query.get("state") or query.get("State") or "").strip()
+    )
+    topic = (query.get("topic") or "").strip().lower()
```

Ruff (0.12.2)

Local variable `topic` is assigned to but never used

Remove assignment to unused variable `topic`

(F841)